### DISCUSSION TITLE
Culling the Bull and Warhammer -> Changing Helltank and Juggernaut to be more unique

### DISCUSSION THREAD

**Discussion by Kami-Interface** [Team-QUIET/QUIET-Community-Edition#127] [Ideas]
Now that the Rommel has replaced the Banshee we're left with 4 tanks in UEF T3 land

Warhammer
Bull
Helltank
Juggernaut

**Proposal**
Remove the Warhammer
**Motivation**
Nobody ever builds this unit. The standard assault tank for UEF should be cheap enough already, and if you need something more affordable, then build a Titan or a T2 unit.

**Proposal**
Remove the Bull
**Motivation**
The Bull has an ugly unit model, fires many missiles, (which can be argued is not part of the UEF identity. No vanilla UEF units fire rockets, it has always been something associated with Cybran) and suffers from loud sounds. Instead, the Helltank can be made land-only (remove amphib), other tweaks can also be made such as removing the rockets. The model can also be scaled up to be in line with the Bull for example. This would also help differentiate the Helltank from the Juggernaut, which as of today are two very similar units.

**Proposal:**
Rework the Helltank slightly to become the UEF standard assault tank
**Motivation**
See above

**Proposal**
Buff the Juggernaut to make it the true T3.5 unit for the UEF
**Motivation**
The Juggernaut today is very similar to the Helltank, and which one you build kind of just depends on who you are as a person more so than anything. The Juggernaut's HP + shield is for some reason lower than the Helltank's HP though. I think the skin shield needs a buff. The unit model should also be scaled up slightly, and perhaps the unit should be given Torpedos or even better Torpedo defence.

The result from these changes should be that UEF gets a new standard assault tank and a more distinctive T3.5 unit that can challenge something like the Dervish, Mogul and similar

**Comment by PurpleMossClump** [comment]
Warhammer
Agreed. Doesn't do anything that makes it stand out. Titan serves the same role. Plus, the model is of very poor quality.

Bull
While I disagree about the model being bad (It's far from hideous, but I do think it just uses existing textures from another unit), I'm not against its removal, provided another unit will be filling the same role as a Main Battle Tank for the UEF. The Bull has been a very reliable, powerful general purpose battle tank. I think it would be missed, but if the Helltank can fill the role, then I see no issue with getting rid of an extra tank.

Helltank
As stated, if it's going to fill the void left by the Bull's absence, I see no issues. 

Juggernaut
Buffs to make it a powerful T3.5 powerhouse sounds interesting. Having each unit stand out from the others is a welcome idea, to me.

**Comment by Azraeel** [comment]
Bull:
I'm personally against the removal of the Bull. The bull is iconic in LOUD for UEF and it provides the ultimate general purpose MBT known in the LOUD Universe. It really is just a solid unit, it may be too loud but I let that slide as it's a well designed unit in what it does, and it's very balanced unit right now.

On the other hand, I also don't think the helltank is a good replacement due to it's design.

**Comment by Kami-Interface** [comment]
@Azraeel so should I close this discussion? I see the Helltank is being reworked into a different unit so I assume you won't do the above

**Comment by Azraeel** [comment]
@Kami-Interface Do not close this, i merged it to see if people physically when they have their hands on it, like the changes.
If they do not we'll rework it to what the community wants again etc etc repeat till most are happy.
